src/pages/NoticeTree/index.tsx

```diff
@@ -49,8 +67,13 @@ const NoticeTree = () => {
 						{username}의 알림나무<span>읽지 않은 열매 {totalUnreadMessageCount} </span>
 					</S.NoticeMainText>
 				</S.NoticeTextWrapper>
-				<Tree readNoticeMessage={readNoticeMessage} messages={noticeMessages ? noticeMessages : null} />
-				<AlertPopUp username={username} messageCount={totalUnreadMessageCount} showAlertMessage={showAlertMessage} />
+				<Tree updateReadMessageHandler={updateReadMessageHandler} messages={noticeMessages ? noticeMessages : null} />
```

Comment:
```suggestion
				<Tree updateReadMessageHandler={updateReadMessageHandler} messages={noticeMessages ?? null} />
```

`nullish coalescing` 으로 가능하지 않을까요 ~?
`noticeMessages` 타입을 확인해봤는데 가능해보여서 말씀드립니다 !